pandas/core/shared_docs.py

```diff
@@ -114,11 +114,17 @@
 as_index : bool, default True
     For aggregated output, return object with group labels as the
```

Comment:
With your addition (if we add in transformations, see below), I think "For aggregated output," can be removed. But I'm also okay with it staying as well.

Reply:
I removed "For aggregated output,". With my addition, there is no need to write this.